Can't start the Vulcan project using the `quick-start-from-nodejs` template

## Environment

- Version:
  <!-- Use command "vulcan version" to get your packages' version -->

  ```
  cli version: 0.4.0
  core version: 0.4.0
  build version: 0.4.0
  serve version: 0.4.0
  ```

- OS: macOS
- node: v16.17.0

## What's wrong?

1. Can't start the Vulcan project using the `quick-start-from-nodejs` template
2. After further inspection using `vulcan serve`,  there was an error message showing: Error  Cannot find module 'bcryptjs'

## What's the correct behavior?

It seems like `bcryptjs` is missing inside the `package.json` file; and at least the server can be successfully started with `"bcryptjs": "^2.4.3"` installed.

## Reproducing steps

1. installed VulcanSQL CLI [from NPM](https://vulcansql.com/docs/get-started/installation#install-from-npm)
2. `vulcan init --template=quick-start-from-nodejs`
3. `vulcan start --watch` -> we can found that the server is not running at localhost:3000
4. `vulcan serve` -> there should be an error message as described above

--- a/package.json
+++ b/package.json
@@ -120,7 +120,6 @@
     "babel-jest": "27.5.1",
     "babel-plugin-import": "^1.13.5",
     "babel-plugin-styled-components": "1.10.7",
-    "bcryptjs": "^2.4.3",
     "commitizen": "^4.2.5",
     "cypress": "^9.1.0",
     "cz-conventional-changelog": "^3.3.0",

